traitsui/editors/list_editor.py

```diff
@@ -85,6 +85,9 @@ class ToolkitEditorFactory(EditorFactory):
     #: Can the list be reorganized, or have items added and deleted.
     mutable = Bool(True)
 
+    #: Should the scrollbars be displayed if the list is too long.
+    scrollable = Bool(True)
```

Comment:
If you declare as:
```
scrollable = Bool(True, sync_value=True)
```
then the value will be copied from the factory to the actual editor automatically.

This is a new-ish feature.

Reply:
This is not working for me -- the value is not being copied from the factory to the editor. I have defined a `scrollable` trait in the editor (set to `True` by default), and removed the `_scrollable_default` method that was copying the value earlier.

Reply:
Hmmm... possibly a bug somewhere, then, but this line should copy things from the factory to the editor: https://github.com/enthought/traitsui/blob/master/traitsui/editor.py#L584